src/components/inputs/money-input/money-input.js

```diff
@@ -2,13 +2,47 @@ import React from 'react';
 import PropTypes from 'prop-types';
 import invariant from 'tiny-invariant';
 import has from 'lodash.has';
+import Select from 'react-select';
+import ClearIndicator from '../../internals/clear-indicator';
+import DropdownIndicator from '../../internals/dropdown-indicator';
 import isNumberish from '../../../utils/is-numberish';
 import filterDataAttributes from '../../../utils/filter-data-attributes';
 import Contraints from '../../constraints';
 import styles from './money-input.mod.css';
 import currencies from './currencies.json';
-import Currency from './currency';
-import CurrencyDropdown from './currency-dropdown';
+import createSelectStyles from '../../internals/create-select-styles';
+import * as vars from '../../../../materials/custom-properties.json';
+
+// overwrite styles of createSelectStyles
+const createCurrencySelectStyles = ({
+  hasWarning,
+  hasError,
+  hasNoCurrencies,
+}) => {
+  const selectStyles = createSelectStyles({ hasWarning, hasError });
+  return {
+    ...selectStyles,
+    control: (base, state) => ({
+      ...selectStyles.control(base, state),
+      borderTopRightRadius: '0',
+      borderBottomRightRadius: '0',
+      borderRight: '0',
+      width: '72px',
```

Comment:
I'm wondering if we can extract this attributes to a external `const commonAttributes` or something like that. I know that you are not using them all over the place but maybe improves the readibility  😋